website/addons/box/model.py

```diff
@@ -188,6 +188,7 @@ def __repr__(self):
 
 class BoxNodeSettings(StorageAddonBase, AddonNodeSettingsBase):
 
+    #foreign_user_settings = fields.ForeignField(
```

Comment:
It needs to be uncommented before migration, and the one below it removed. Commented out because tests will fail otherwise.